### PR TITLE
Adding COC, contributing doc

### PR DESCRIPTION
**Pull request by psschwei**
# Changes

- Adding Code of Conduct, Contribution docs as part of [GA](https://github.com/knative/community/issues/680)

/kind documentation

/assign @houshengbo

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+confidentiality with regard to the reporter of an incident. Further details of
+specific enforcement policies may be posted separately.
+
+TOC members who do not follow or enforce the Code of Conduct in good faith may
+face temporary or permanent repercussions as determined by other members of the
+project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
```
Comment (markusthoemmes) [inline CODE-OF-CONDUCT.md:72]: Broken link

Comment (houshengbo) [reply CODE-OF-CONDUCT.md:72]: Should be OK, can be verified here: https://github.com/psschwei/operator/blob/ga-docs/CODE-OF-CONDUCT.md
[homepage]: https://www.contributor-covenant.org defines the homepage link.

Comment (psschwei) [reply CODE-OF-CONDUCT.md:72]: updated

**Comment by evankanderson** [comment]
(If these are copies of the doc in https://github.com/knative/community, it should be possible to add to https://github.com/knative-sandbox/knobots/blob/main/actions/update-community/entrypoint.sh to copy additional files to all repos.)

**Pull request merged**